packages/slug/slug.1.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+authors: "Khoa Nguyen"
+homepage: "https://github.com/thangngoc89/ocaml-slug"
+maintainer: "hi@khoanguyen.me"
+dev-repo: "git+ssh://git@github.com:thangngoc89/ocaml-slug.git"
+bug-reports: "https://github.com/thangngoc89/ocaml-slug/issues"
+license: "MIT"
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "2.0"}
+  "opam-lock" {dev}
```

Comment:
why is opam-lock required in dev mode? I can’t find any mention of it in the source code

Reply:
I have no ideas. I copied this from 1.0.0. Let’s me delete this